receiver/mongodbatlasreceiver/receiver.go

```diff
@@ -231,3 +243,105 @@ func (s *receiver) extractProcessDiskMetrics(
 	}
 	return nil
 }
+
+// Log receiver logic
+func (s *receiver) Start(ctx context.Context, host component.Host) error {
+	go func() {
+		s.KickoffReceiver(ctx)
+	}()
+	return nil
+}
+
+func (s *receiver) Shutdown(ctx context.Context) error {
+	for _, stopperChan := range s.stopperChanList {
+		close(stopperChan)
+	}
+	ctx.Done()
```

Comment:
ctx.Done() returns a channel that signals when the context is cancelled, this is a no-op.
```suggestion
```